### ISSUE TITLE
Compilation failure due to missing dependecy

### ISSUE THREAD

**Issue by rtobar** [ska-sa/spead2#240]
While trying to compile spead (both standalone or as a python package), `configure` detects ibverbs support, but compilation still fails:

```
[...]
Libraries:

    ibverbs: yes
    ibverbs HW rate limit: yes
[...]

In file included from src/common_loader_rdmacm.cpp:30:
include/spead2/common_loader_rdmacm.h:33:10: fatal error: rdma/rdma_cma.h: No such file or directory
   33 | #include <rdma/rdma_cma.h>

```

When compiling standalone I can easily bypass this by running `./configure --without-ibv`, in which case compilation continues. Compiling the python package requires modifying `setup.py` to pass the flag to `configure` though.

I'm running in Ubuntu 22.10 with `libibverbs-dev` version 42.0-1 installed. If I install `librdmacm-dev` the problem also goes away.

**Comment by bmerry** [comment]
Ok, so I guess I need to check for that header as well before enabling the ibverbs support. Since there is an easy workaround (install librdmacm-dev) I'll leave this open for now, since I'm looking into replacing the whole build system.

**Comment by bmerry** [comment]
@rtobar I think this should be fixed in 4.0.0b1. Let me know if not.
